test/e2e/yarn-pnp/test/utils.ts

```diff
@@ -31,7 +31,7 @@ export function runTests(example = '') {
           prev.push(`${cur}@${dependencies[cur]}`)
           return prev
         }, [] as string[])
-        return `yarn set version berry && yarn config set enableGlobalCache true && yarn config set compressionLevel 0 && yarn add ${pkgs.join(
+        return `yarn set version 3.1.1 && yarn config set enableGlobalCache true && yarn config set compressionLevel 0 && yarn add ${pkgs.join(
```

Comment:
```suggestion
        return `yarn set version berry && yarn config set enableGlobalCache true && yarn config set compressionLevel 0 && yarn config set pnpEnableEsmLoader false && yarn add ${pkgs.join(
```

Reply:
I don't think we can set this unless it's the default for users with yarn pnp, we want to test the default experience users will be having